taxonomies/ingredients.txt

```diff
@@ -31810,6 +31835,10 @@ fi:Luontainen hiiva
 fr:levure naturelle
 it:naturale-fermento, lievito naturale
 
+<en:yeast
+en:organic yeast
+de:Biologische Hefe
+
```

Comment:
This should be instead picked up as a label by the ingredients parser. Add `Biologische` as a synonym here: https://github.com/openfoodfacts/openfoodfacts-server/blob/49dea456c786a8adac8b3a8eb371a0dc1413a132/taxonomies/labels.txt#L2902

Reply:
Okay thanks, I saw it that way at other points but it would be tedious  to make that for every ingredient.